src/containers/App.jsx

```diff
@@ -135,7 +135,7 @@ const useStyles = makeStyles(theme => ({
     position: 'absolute',
     right: 0,
     width: theme.spacing(40),
-    zIndex: '100',
+    zIndex: 1201,
```

Comment:
This seems like a pretty large z-index, any particular reason why? Also, should it numeric or a string?

Reply:
It needs to be higher than the z-index for the sidebar panel, which is 1200 (default Material UI classes)! I think either a number or string works but I made it a number to be consistent with the other CSS rules.